src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -0,0 +1,1259 @@
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  days_to_comorbidity:
+    $ref:
+      - "_terms.yaml#/days_to_comorbidity/common"
+    type: integer
+    maximum: 32872
+    minimum: -32872
+
+  diabetes_treatment_type:
+    $ref:
+      - "_terms.yaml#/diabetes_treatment_type/common"
+    enum:
+      - Alpha-Glucosidase Inhibitor
+      - Biguanide
+      - Diet
+      - Injected Insulin
+      - Insulin
+      - Linagliptin
+      - Oral Hypoglycemic
+      - Sulfonylurea
+      - Thiazolidinedione
+      - Other
+      - Unknown
+      - Not Reported
+    enumDef:
+      Other:
+        $ref:
+          - "_terms_enum.yaml#/other/common"
+      Unknown:
+        $ref:
+          - "_terms_enum.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  dlco_ref_predictive_percent:
+    $ref:
+      - "_terms.yaml#/dlco_ref_predictive_percent/common"
+    type: number
+    maximum: 100
+    minimum: 0
+
+  exercise_frequency_weekly:
+    description: >-
+      Text term that indicates the number of days that exercise occurred in the past
+      seven days.
+    enum:
+      - 5 or more days
+      - 3 to 4 days
+      - 1 to 2 days
+      - None
+    enumDef:
+      None:
+        $ref:
+          - "_terms_enum.yaml#/none/common"
+
+  eye_color:
+    $ref:
+      - "_terms.yaml#/eye_color/common"
+    enum:
+      - Amber
+      - Blue
+      - Brown
+      - Gray
+      - Green
+      - Hazel
+      - Red & Violet
+      - Other
+      - Not Reported
+    enumDef:
+      Other:
+        $ref:
+          - "_terms_enum.yaml#/other/common"
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  fev1_fvc_post_bronch_percent:
+    $ref:
+      - "_terms.yaml#/fev1_fvc_post_bronch_percent/common"
+    type: number
+    maximum: 100
+    minimum: 0
+
+  fev1_fvc_pre_bronch_percent:
+    $ref:
+      - "_terms.yaml#/fev1_fvc_pre_bronch_percent/common"
+    type: number
+    maximum: 100
+    minimum: 0
+
+  fev1_ref_post_bronch_percent:
+    $ref:
+      - "_terms.yaml#/fev1_ref_post_bronch_percent/common"
+    type: number
+    maximum: 100
+    minimum: 0
+
+  fev1_ref_pre_bronch_percent:
+    $ref:
+      - "_terms.yaml#/fev1_ref_pre_bronch_percent/common"
+    type: number
+    maximum: 100
+    minimum: 0
+
+  haart_treatment_indicator:
+    $ref:
+      - "_terms.yaml#/haart_treatment_indicator/common"
+    enum:
+      - "Yes"
+      - "No"
+      - Unknown
+      - Not Reported
+    enumDef:
+      "Yes":
+        $ref:
+          - "_terms_enum.yaml#/yes/common"
+      "No":
+        $ref:
+          - "_terms_enum.yaml#/no/common"
+      Unknown:
+        $ref:
+          - "_terms_enum.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  height:
+    $ref:
+      - "_terms.yaml#/height/common"
+    type: number
+    minimum: 0
```

Comment:
zero?

Also, should units be included? (cm, ft) unless it's standardized already (cm, mm, or m)

Reply:
Yes, we define height in dictionary viewer as "The height of the patient in centimeters." 

minimum of 0 is a placeholder to users do not accidentally submit negative values I believe. Also we have pediatric cancer data, so defining a minimum above 0 may get tricky, not sure what the minimum length of a baby should be.